### DISCUSSION TITLE
Looking forward to announce redesign of GameZone  📢

### DISCUSSION THREAD

**Discussion by kunjgit** [kunjgit/GameZone#5175] [Announcements]
checkout #5174 !
creative designs are welcomed from GameZone famlily ❤️ !

Thank you

**Comment by dishamodi0910** [comment]
Are there any sample designs in your mind that could be implemented for Gamezone? @kunjgit

**Reply by kunjgit** [reply]
Not finalized one but as far as we get some from community we can adjust those changes there and scale it up 😄✨

@dishamodi0910
